tests/conftest.py

```diff
@@ -404,6 +430,16 @@ def lambda_handler(event, context):
         zip_content = zip_output.read()
 
     client = boto3.client("lambda", region_name=region_name)
+    vpc_config = {
+        "SubnetIds": [],
+        "SecurityGroupIds": []
+    }
+    if security_groups is not None:
+        vpc_config["SubnetIds"] = vpc_config["SubnetIds"] + subnet_ids
+    if security_groups is not None:
+        vpc_config["SecurityGroupIds"] = \
+            vpc_config["SecurityGroupIds"] + security_groups
```

Comment:
```suggestion
    if security_groups is not None:
        vpc_config["SubnetIds"] = subnet_ids
    if security_groups is not None:
        vpc_config["SecurityGroupIds"] = security_groups
```

Why do you append? Can't you just assign the field directly?

Reply:
yes you're right.